_data/glossary.yml

```diff
@@ -1,9 +1,79 @@
+- keyword: ITK
+  expansion: Interoperability Tool Kit
+  definition: A collection of interoperability standards and tools produced by NHS Digital to support interoperability in healthcare.
+- keyword: IUC CAS 
+  expansion: Integrated Urgent Care Clinical Assessment Service
+  definition: Physical and/or virtual hub providing clinical advice to patients contacting the 111 or 999 services, as well as providing clinical support to clinicians (particularly ambulance staff such as paramedics and emergency technicians) to ensure that no decision is made in isolation. Also known as Care Advice Service or Clinical Assessment Service, previously known as a Clinical Navigation Hub.
+- keyword: Patient 
+  definition: Generic term used in Commissioning Standards for citizens / public using the service.
+- keyword: PMPR  
+  expansion: Predictive Modelling of Patient Risk
+  definition: Using historical data to identify the patient-level variables that predict over-triage and under-triage for different Pathways or symptom groups. These patient-level variables can then be used to identify future service users who are at high risk of over- and under-triage.
+- keyword: POC 
+  expansion: Proof of Concept
+  definition: A proof of concept is a demonstration, the purpose of which is to verify that certain concepts or theories have the potential for real-world application. POC is therefore a prototype that is designed to determine feasibility, but does not represent deliverables.
+- keyword: Provider 
+  definition: Entity contracted by a commissioner to provide a service
+- keyword: SNOMED CT 
+  expansion: Systemised Nomenclature of Medicine (Clinical Terms). 
+  definition: A structured clinical vocabulary for use in an electronic health record. It is the most comprehensive and precise clinical health terminology product in the world.
+- keyword: Supplier 
+  definition: An entity providing a system or system components to Providers or the System user Person interacting with the system
+- keyword: Triage 
+  definition: The process of determining the priority of patients' treatments based on the severity of their condition
+- keyword: UEC
+  expansion: Urgent and Emergency Care 
+  definition: An area of Health Care services that primarily deals with urgent same day outpatient contacts for patient needs such as minor illness and injuries, and involves sign-posting and treatment of patients in settings such as 111, 999, OOH (Out Of Hours) GPs, A&E and minor injuries units, or through arrangement of urgent appointments (GP and Dental).
+- keyword: UTC
+  expansion: Urgent Treatment Centres
+  definition: Medical treatment facility where patients can be treated for minor emergencies and injury.
+- keyword: WIC
+  expansion: Walk-in centre
+  definition: NHS walk-in centres offer convenient access to a range of treatments. WICs are managed by Clinical Commissioning Groups (CCGs), dealing with minor illnesses and injuries
```

Comment:
You should reference that WICs will eventually be replaced by UTCs.

Reply:
Amended slightly:
NHS walk-in centres offer convenient access to a range of treatments. WICs are managed by Clinical Commissioning Groups (CCGs), dealing with minor illnesses and injuries. Walk-in centres are gradually being replaced by urgent treatment centres (UTCs).

Beefed up UTC description:

A medical treatment facility where patients can be treated for minor emergencies and injury. UTCs work alongside other parts of the urgent care network including primary care, community pharmacists, ambulance and other community-based services to provide a locally accessible and convenient alternative to A&E for patients who do not need to attend hospital. Commonality of service provision under the UTC name will increasingly end the current confusing range of options and simplify the system so patients know where to go and have clarity of which services are on offer wherever they are in the country.